test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/ZookeeperK8sService.java

```diff
@@ -161,6 +162,9 @@ private V1beta1CustomResourceDefinition getZKOperatorCRD() {
                                                      .build())
                                   .withScope("Namespaced")
                                   .withVersion(CUSTOM_RESOURCE_VERSION)
+                                  .withNewSubresources()
+                                                     .withStatus(new V1beta1CustomResourceDefinitionStatus())
```

Comment:
Can you fix the indentation here, please?

Reply:
Fixed.